tests/test_check_parallel.py

```diff
@@ -0,0 +1,153 @@
+import os
+import shutil
+import unittest
+from base import *
+
+
+class TestCheckParallel(TestZstash):
+    """
+    Test `zstash check` in parallel.
+    """
+    # `zstash check` is tested in TestCheck and TestCheckParallel.
+    # x = on, no mark = off, b = both on and off tested
+    # option | Check | CheckMismatch | CheckParallelKeepTars | CheckParallel | CheckParallelVerboseMismatch | KeepTarsWithPreviouslySetHPSS |
```

Comment:
Change `KeepTarsWithPreviouslySetHPSS` to `CheckParallelKeepTars`. Also update in `test_check.py`.